spec/requests/api/v3/courses/index_spec.rb

```diff
@@ -208,10 +191,9 @@
           json_response = JSON.parse(response.body)
           course_ids << json_response.dig("data", 0, "id")
 
-          get page_3
-
-          json_response = JSON.parse(response.body)
-          course_ids << json_response.dig("data", 0, "id")
+          expect {
+            get page_3
+          }.to raise_error(Pagy::OverflowError)
```

Comment:
So when something is out of bounds it raises error...

Maybe add it here as well?
https://github.com/DFE-Digital/teacher-training-api/blob/66fc56992741bb605b9dd932505002cc1ec2dc0c/config/application.rb#L38-L42

Reply:
done